barretenberg/cpp/src/barretenberg/vm/generated/AvmMini_arithmetic.test.cpp

```diff
@@ -0,0 +1,446 @@
+#include "barretenberg/ecc/curves/bn254/fr.hpp"
```

Comment:
This should move out of the generated folder into a /test/ folder

Reply:
Done.